apps/docs/pages/guides/self-hosting/docker.mdx

```diff
@@ -62,87 +107,94 @@ Update the `.env` file with your own secrets. In particular, these are required:
 - `SITE_URL`: the base URL of your site.
 - `SMTP_*`: mail server credentials. You can use any SMTP server.
 
-### Adding a firewall
```

Comment:
not actually required - I found out the Docker-compose updates iptables automatically (was that always the case)? So now the ports exposed in the docker compose file are accessible (only port 8000), and the rest are not

Reply:
Yeah Docker adds their own [iptables chains](https://docs.docker.com/network/packet-filtering-firewalls/) which bypasses other firewalls [like ufw](https://docs.docker.com/network/packet-filtering-firewalls/#docker-and-ufw). Still good practice to have a firewall, but probably out of scope for this guide.